centaur/src/main/scala/centaur/test/metadata/WorkflowFlatMetadata.scala

```diff
@@ -35,7 +40,26 @@ case class WorkflowFlatMetadata(value: Map[String, JsValue]) extends AnyVal {
     mismatchErrors ++ missingErrors
   }
 
+  def diff(actual: WorkflowFlatMetadata): Iterable[String] = {
```

Comment:
If we use diffson to generate full-json diffs, we probably don't need this function anymore either!